Issue 1241: opentelemtry attribute typing error

wmolde opened the pull request:
fixes #1241

Updating the opentelementry attribute definition for `SERVER_PORT` from a `stringKey` to the type appropriate `longKey`.

baywet reviewed (changes requested):
Thanks for the contributions!
Can you also please:
- bump the patch version in gradle.properties
- add a changelog entry in the changelog (today's date, changed)

wmolde commented:
@microsoft-github-policy-service agree company="Kollective Technologies"

wmolde commented:
> Thanks for the contributions! Can you also please:
> 
> * bump the patch version in gradle.properties
> * add a changelog entry in the changelog (today's date, changed)

Updated with the requested changes.
